docs/CHANGELOG.md

```diff
@@ -96,7 +96,8 @@ const User = new GraphQLObjectType({
           sqlPaginate: true,
           sortKey: [
             { column: 'created_at', direction: 'desc' },
-            { column: 'id', order: 'desc' }
+            { column: 'id', order: 'desc' },
+            { column: 'json', key: '"json"->\'field\'' order: 'desc' }
```

Comment:
Can it be a string or a function? Or just a function? You can use join monster's `unthunk` for accepting both if you like

Reply:
My bad, but nice catch! I forgot to update after switching to a function this morning 🙈